docs/python/testing.md

```diff
@@ -12,144 +12,27 @@ MetaSocialImage: images/tutorial/python-social.png
 
 The [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) supports testing with Python's built-in [unittest](https://docs.python.org/3/library/unittest.html) framework and [pytest](https://docs.pytest.org/).
```

Comment:
Provide a high-level description of what the extension brings: it builds on the built-in testing features in VS Code and provides test discovery, test coverage, and running and debugging tests.